soh/soh/Enhancements/randomizer/3drando/spoiler_log.cpp

```diff
@@ -99,12 +99,12 @@ void WriteIngameSpoilerLog() {
         stringOffsetMap; // Map of strings to their offset into spoiler string data array
     stringOffsetMap.reserve(ctx->allLocations.size() * 2);
 
-    // Sort all locations by their group, so the in-game log can show a group of items by simply starting/ending at
+    // Sort all locations by their area, so the in-game log can show a area of items by simply starting/ending at
     // certain indices
     std::stable_sort(ctx->allLocations.begin(), ctx->allLocations.end(), [](const RandomizerCheck& a, const RandomizerCheck& b) {
-        auto groupA = Rando::StaticData::GetLocation(a)->GetCollectionCheckGroup();
-        auto groupB = Rando::StaticData::GetLocation(b)->GetCollectionCheckGroup();
-        return groupA < groupB;
+        RandomizerCheckArea areaA = Rando::StaticData::GetLocation(a)->GetArea();
+        RandomizerCheckArea areaB = Rando::StaticData::GetLocation(b)->GetArea();
+        return areaA < areaB;
```

Comment:
How comparable were RCA and CCG in this regard?

Reply:
Pretty similar except for some small differences like dmt & dmc being together (iirc market + tot + castle grounds were also combined into 1).

Reply:
So, SCCG actually had fewer groups than RCA does? I can live with that, as long as it doesn't affect other parts of logic. @Pepper0ni would that change anything about hint generation, fundamentally?

Reply:
I literally deleted this whole function in my shop unification as it's an unused 3DS feature.

Reply:
Right, but this isn't the only place RCCG was replaced by RCA. This was just where I first saw it.